### ISSUE TITLE
Add support for Binance lending assets

### ISSUE THREAD

**Issue by coblee** [rotki/rotki#1502]
## Abstract

When using the Binance API to fetch coin balances, it shows an error:
`Found unknown binance asset LDBNB. Ignoring its balance query.`

## Motivation

I believe Binance lending assets all start with `LD`. So you can just remove the `LD` part of the symbol and add to the balance for that coin. So `LDBNB` is `BNB` and `LDUSDT` is `USDT`

It would be useful to show that this is from Binance lending/savings, but it's not that important.

## Specification

If the feature is techical in nature please write as detailed as possible a specification of what needs to be built.

**Comment by LefterisJP** [comment]
Oh that's interesting. If it's that simple we could even sneak it in v1.8. Sounds like the way we handle kraken staked assets.
https://feedback.coinstats.app/b/coins-token-requests/binance-wallets/

Checking their API, there is more we could do, like show what is in the savings wallet and what's the interestrate. But that's for later: https://binance-docs.github.io/apidocs/spot/en/#savings-endpoints

```
[
    {
        "asset": "BTC",
        "avgAnnualInterestRate": "0.00250025",
        "canPurchase": true,
        "canRedeem": true,
        "dailyInterestPerThousand": "0.00685000",
        "featured": true,
        "minPurchaseAmount": "0.01000000",
        "productId": "BTC001",
        "purchasedAmount": "16.32467016",
        "status": "PURCHASING",
        "upLimit": "200.00000000",
        "upLimitPerUser": "5.00000000"
    },
    {
        "asset": "BUSD",
        "avgAnnualInterestRate": "0.01228590",
        "canPurchase": true,
        "canRedeem": true,
        "dailyInterestPerThousand": "0.03836000",
        "featured": true,
        "minPurchaseAmount": "0.10000000",
        "productId": "BUSD001",
        "purchasedAmount": "10.38932339",
        "status": "PURCHASING",
        "upLimit": "100000.00000000",
        "upLimitPerUser": "50000.00000000"
    }
]
```

**Comment by LefterisJP** [comment]
Closed by #1503. More similar work to be done in https://github.com/rotki/rotki/issues/1505

**Comment by AkioKiyota** [comment]
hi... sorry for bothering but i dont understand whats going on.

here is my usdt balance on my webpage
![resim](https://github.com/rotki/rotki/assets/92454444/87170a9c-6211-4028-8fa7-daa1b1659232)
![resim](https://github.com/rotki/rotki/assets/92454444/0318a2ce-6d6c-4325-b3b3-95d956254775)

and here is the balance i see on my demo website which i used python-binance package 
![resim](https://github.com/rotki/rotki/assets/92454444/0af5d62a-bb28-4c7b-a850-e14f0f5a3eab)
![resim](https://github.com/rotki/rotki/assets/92454444/19b6a0fd-407d-4e6b-9860-b4e090780095)

its showing me ldusdt instead of usdt for some reason and i dont really get it???
